src/app/course/event/event-question-view/event-question-view.component.scss

```diff
@@ -24,3 +24,11 @@
         flex-grow: 1;
     }
 }
+
+.box {
+    height: calc(80vh - 12rem - 2px); // Window height - (header + footer + wiggle room) - border width
+}
+
+.fill-space {
+    width: 22vh;
+}
```

Comment:
have you used this?

Reply:
Line 43 of event-question-view.component.html, it is what causes the question name to push the open/close button into its proper place